packages/components/src/components/post-tooltip/post-tooltip.tsx

```diff
@@ -0,0 +1,231 @@
+import { Component, h, Element, Prop, Host, Method } from '@stencil/core';
+import {
+  computePosition,
+  Placement,
+  flip,
+  shift,
+  autoUpdate,
+  offset,
+  inline,
+  arrow,
+} from '@floating-ui/dom';
+import isFocusable from 'ally.js/esm/is/focusable';
+
+// Polyfill for popovers, can be removed when https://caniuse.com/?search=popover is green
+import '@oddbird/popover-polyfill';
+
+// Patch for long press on touch devices
+import 'long-press-event';
+
+import { version } from '../../../package.json';
+
+const sidemap = {
+  top: 'bottom',
+  right: 'left',
+  bottom: 'top',
+  left: 'right',
+};
+
+interface IPopoverElement {
+  showPopover: () => void;
+  hidePopover: () => void;
+  togglePopover: (force?: boolean) => boolean;
+}
+
+@Component({
+  tag: 'post-tooltip',
+  styleUrl: 'post-tooltip.scss',
+  shadow: true,
+})
+export class PostTooltip {
+  @Element() host: HTMLPostTooltipElement;
+
+  /**
+   * Define the placement of the tooltip according to the floating-ui options available at https://floating-ui.com/docs/computePosition#placement.
+   * Tooltips are automatically flipped to the opposite side if there is not enough available space and are shifted
+   * towards the viewport if they would overlap edge boundaries.
+   */
+  @Prop() readonly placement?: Placement = 'top';
+
+  /**
+   * Internally used to track changes to the class attribute on the host element
+   */
+  @Prop() readonly class: string;
+
+  private tooltipRef: HTMLElement & IPopoverElement;
+  private arrowRef: HTMLElement;
+  private trigger: HTMLElement;
+  private clearAutoupdate: () => void;
+
+  connectedCallback() {
+    if (!this.host.id) {
+      throw new Error(
+        'No id set: <post-tooltip> must have an id, linking it to it\'s target element using the data-tooltip-target attribute.',
+      );
+    }
+
+    this.trigger = document.querySelector(`[data-tooltip-target="${this.host.id}"]`);
```

Comment:
For the same tooltip? Highly unlikely IMO. Also, it's possible to trigger it programmatically from anywhere.

Reply:
What if you have a table with many rows and in each row is a info icon which will show you (for example) one of three diffrent static tooltips? If you could specify those three tooltips only once and add as much triggers to shos/hide them as you like, this could be a nice way to reduce code.
Otherwhise you would need to define a tooltip for every row or am I missing something?

Reply:
I update the component and docs to support multiple triggers